### PR TITLE
Add snmp plugin

### PR DESCRIPTION
**Pull request by titilambert**
This is the new PR of #446

### CODE REVIEW

File: plugins/snmp/snmp.go [hunk 1]
```diff
@@ -0,0 +1,348 @@
...
+					// TODO: is data tag usefull ???
+					if oid_name != "" {
+						tags["data"] = oid_name
+					} else if oid.Oid == variable.Name {
+						tags["data"] = oid.Name
+					} else {
+						tags["data"] = variable.Name
+					}
+					tags["host"], _, _ = net.SplitHostPort(h.Address)
+					acc.Add(tags["data"], gosnmp.ToBigInt(variable.Value), tags)
```
Comment (titilambert) [inline plugins/snmp/snmp.go]: Do I need to change those line by 
```
                    var field_name string
                    if oid_name != "" {
                        field_name = oid_name
                    } else if oid.Oid == variable.Name {
                        field_name = oid.Name
                    } else {
                        field_name = variable.Name
                    }
                    tags["host"], _, _ = net.SplitHostPort(h.Address)
                    var fields map[string]interface{}
                    fields[field_name] = variable.Value
                    acc.AddFields(field_name, fields, tags)
```

???

Comment (sparrc) [inline plugins/snmp/snmp.go]: yes, please add as fields

**Comment by titilambert** [comment]
@sparrc Do you prefer I squash all commits ?

**Comment by titilambert** [comment]
@sparrc Hello !
I added tests, I squashed all commits, I signed CLA, I think this one is ready :)

**Comment by titilambert** [comment]
I just saw that sample configuration is bad I'm fixing it and I'm adding your suggestion about https://github.com/paulstuart/influxsnmp.

**Comment by titilambert** [comment]
@sparrc 
Should I create: 
snmp_ifOutTraffic,host=127.0.0.1,instance=2 ifOutTraffic=60i 1450807658010149028
snmp_ifOutTraffic,host=127.0.0.1,instance=3 ifOutTraffic=99i 1450807658010149028
or
snmp_ifOutTraffic,host=127.0.0.1 ifOutTraffic.3=99i ifOutTraffic.2=60i 1450807658010149028
or
????

**Comment by sparrc** [comment]
thanks @titilambert, I would go with the former

**Comment by titilambert** [comment]
@sparrc which former one ? Could you write it down, just to be explicit ? Thanks !

**Comment by sparrc** [comment]
@titilambert, this one:

snmp_ifOutTraffic,host=127.0.0.1,instance=2 ifOutTraffic=60i 1450807658010149028
snmp_ifOutTraffic,host=127.0.0.1,instance=3 ifOutTraffic=99i 1450807658010149028

**Comment by titilambert** [comment]
@sparrc Perfect I done this one :)

**Comment by titilambert** [comment]
I close this PR to open a new one based on 0.3.0

**Pull request closed without merging**